common/lib/container-definitions/api-extractor.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+    "extends": "@fluidframework/build-common/api-extractor-common-report.json",
+    "apiReport": {
+      "enabled": true,
+      "reportFolder": "<projectFolder>../api-report/"
```

Comment:
```suggestion
      "reportFolder": "<projectFolder>/api-report/"
```

Reply:
i was trying to decide where these should go. what's the strategy here?

Reply:
I've been trying to output all the API reports for a given "mono-repo" (workspace?) to a single folder relative to the workspace root. Since these packages are all independent, I suggest we put the reports directly within the projects. (I'm thinking about this for the API extraction process, too, but those files need to be aggregated as part of the build so it's more complicated.)

This should make us able to take a workspace and break it out into its own repo without any changes to the API pipeline.

Reply:
Also why can't you just read my mind? 😝